shortstories/tests.py

```diff
@@ -32,3 +32,10 @@ def  test_current_shortstory(self):
         current_shortstory = create_shortstories(author=user_id, status='p', title ="Villete", body="A romance written by Bronte Charlotte", publication_date='2000-08-25')
         response = self.client.get(reverse('shortstories:detail', args=(current_shortstory.id,)))
         self.assertContains(response, current_shortstory.title)
+
+
+class shortstoriesfunction(TestCase):
+    def test_snippet_function(self):
```

Comment:
Can you add one more test?
Please also the following style naming your class and function
https://docs.djangoproject.com/en/2.1/topics/testing/overview/